recipes/sdl_ttf/all/conanfile.py

```diff
@@ -97,6 +117,8 @@ def package_info(self):
         self.cpp_info.includedirs.append(os.path.join("include", "SDL2"))
         self.cpp_info.libs = ["SDL2_ttf"]
```

Comment:
Static MSVC has a -static suffix.

Reply:
@madebr 
I understand that -static is not attached to the name of the library, but -static is attached to the target name of CMake.

For this reason, I am trying to fix the same point above.
If I am wrong, I would appreciate it if you could point it out again.

Reply:
Sorry for the troubles I caused for this release!